schemas/test/en/test_validation_sum_against_total_calculated_summary.json

```diff
@@ -0,0 +1,160 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
```

Comment:
its also possible to replicate the error by adding a single Calculated Summary of breakdown 1-4 to the end of `test_validation_sum_against_value_source` which has a similar setup, which might be preferable to the extra schema, but could also detract from what it is actually there to test, not too fussed either way, just an idea if you did want to avoid the extra one, otherwise feel free to resolve